src/backend/dx12/src/command.rs

```diff
@@ -1213,17 +1244,128 @@ impl com::RawCommandBuffer<Backend> for CommandBuffer {
 
     fn blit_image<T>(
         &mut self,
-        _src: &n::Image,
+        src: &n::Image,
         _src_layout: image::Layout,
-        _dst: &n::Image,
+        dst: &n::Image,
         _dst_layout: image::Layout,
-        _filter: image::Filter,
-        _regions: T,
+        filter: image::Filter,
+        regions: T,
     ) where
         T: IntoIterator,
         T::Item: Borrow<com::ImageBlit>
     {
-        unimplemented!()
+        // TODO: only supporting 2D images
+
+        let rtv_pool = {
```

Comment:
looks like it could be owned by the command pool?

Reply:
yes, temporarily owned. Waiting on renderdoc support to drop it immediately.